src/main/java/uk/nhs/digital/nhsconnect/nhais/translator/acceptance/AcceptanceFirstTranslator.java

```diff
@@ -37,51 +46,70 @@
     private final AcceptanceTypeMapper acceptanceTypeMapper;
     private final AcceptanceDateMapper acceptanceDateMapper;
     private final PersonNameMapper personNameMapper;
+    private final PersonPlaceOfBirthMapper personPlaceOfBirthMapper;
+    private final PersonPreviousNameMapper personPreviousNameMapper;
     private final PersonSexMapper personSexMapper;
     private final PersonAddressMapper personAddressMapper;
+    private final PersonDateOfBirthMapper personDateOfBirthMapper;
+    private final DrugsMarkerMapper drugsMarkerMapper;
+    private final FreeTextMapper freeTextMapper;
+    private final ResidentialInstituteNameAndAddressMapper residentialInstituteNameAndAddressMapper;
+    private final OptionalInputValidator validator;
 
     @Override
     public List<Segment> translate(Parameters parameters) throws FhirValidationException {
-//        //TODO: enforce place of birth mandatory when NHS number is missing
```

Comment:
I've added https://gpitbjss.atlassian.net/browse/NIAD-384 to the sprint with zero points so this AC gets tested